src/pages/workspaces/workspace/analysis/modals/modalStyles.ts

```diff
@@ -0,0 +1,12 @@
+import colors from 'src/libs/colors'
+
+//I re-create compute styles here because they're not typed in their native file
```

Comment:
It would be nice to change other files to use this instead of duplicating it.

Bonus points for doing reorganization in a separate PR (or at least commit) to make things easier to review.

Reply:
I'm not sure what you mean by this, I've edited all files that use computestyles to use this iteration of it, and removed the original

Reply:
I see changes to use this instance instead of the old instance in runtime-common-components, but I don't see the removal of that old instance. If the intent was to remove the old version, then I find the comment about "re-create" here confusing.

Reply:
I'll remove it in my next push, thanks for the save

Reply:
For comments like this, that are related to the change itself and might not make sense when looking at the file outside of the context of the change, I think using GitHub comments on the PR can be better than comments in the code. Like once this change is merged, someone looking at this code probably wouldn't understand what "their native file" was referring to.

Reply:
ok, coming right up!